src/blockchain/ain/jellyfish/utxo-provider.service.ts

```diff
@@ -155,6 +158,30 @@ export class UtxoProviderService {
     return neededUnspent;
   }
 
+  private static provideUntilAmountPlusFee(
+    unspent: AddressUnspent[],
+    amountPlusFeeBuffer: BigNumber,
+    sizePriority: UtxoSizePriority,
+  ): [AddressUnspent[], BigNumber] {
+    const neededUnspent: AddressUnspent[] = [];
+    let total = new BigNumber(0);
+    let unspentToSearch: AddressUnspent[] = [];
+    unspent = unspent.sort((a, b) =>
+      sizePriority === UtxoSizePriority.BIG ? this.orderDescending(a, b) : this.orderAscending(a, b),
+    );
```

Comment:
Just curious, would this work?
```ts
unspent = unspent.sort(UtxoSizePriority.BIG ? this.orderDescending : this.orderAscending);
```

Reply:
yes, looking forward to have this stored in my brain, to improve code quality and make it easier for you to review my code...